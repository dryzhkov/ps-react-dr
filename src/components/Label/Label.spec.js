import React from 'react';
import renderer from 'react-test-renderer';
import Label from './Label';

describe('Label', () => {
    test('shows red asterisk when required is enabled', () => {
        const tree = renderer.create(<Label htmlFor="test" label="test" required />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});