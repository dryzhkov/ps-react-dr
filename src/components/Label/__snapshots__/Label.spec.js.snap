// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label shows red asterisk when required is enabled 1`] = `
<label
  htmlFor="test"
  style={
    Object {
      "display": "block",
    }
  }
>
  test
   
  <span
    style={
      Object {
        "color": "red",
      }
    }
  >
     *
  </span>
</label>
`;
